src/EditorFeatures/CSharpTest/Diagnostics/NamingStyles/EditorConfigNamingStyleParserTests.cs

```diff
@@ -12,6 +13,24 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics.NamingStyle
 {
     public class EditorConfigNamingStyleParserTests
     {
+        internal static SymbolKindOrTypeKind ToSymbolKindOrTypeKind(object symbolOrTypeKind)
+        {
+            switch (symbolOrTypeKind)
+            {
+            case TypeKind typeKind:
```

Comment:
Looks like your indentation settings might be off (I think it's usually one deeper)

Reply:
Same comment as https://github.com/dotnet/roslyn/pull/26589#discussion_r193532991

Reply:
I'll update as a separate PR so we don't have to retest this one for just indentation.

Reply:
:arrow_right: Had to rerun tests for retargeting, so fixed in 32e150c9ae85d9ad3e3233ea374e3ae0cab26d3b